src/platform/monitoring/DowntimeNotification/components/Wrapper.jsx

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
 
-export default function DowntimeNotificationWrapper({ status, children }) {
+export default function DowntimeNotificationWrapper({
+  status,
+  children,
+  className,
+}) {
+  const classes = `downtime-notification ${className}`;
```

Comment:
We need to be able to override the `row-padded` class that has always been applied here but also need to make sure we don't introduce any regressions. Since this Wrapper component is only used by the DowntimeApproaching component we can make sure that the `row-padded` is passed in by default via the parent DowntimeApproaching component. But now any component that uses a DowntimeApproaching component can customize that behavior.